src/timezone_impl.rs

```diff
@@ -77,6 +77,40 @@ pub trait OffsetComponents {
     fn dst_offset(&self) -> Duration;
 }
 
+/// Timezone offset name information.
+///
+/// This trait exposes display names that describe an offset in
+/// various situations.
+///
+/// ```
+/// # extern crate chrono;
+/// # extern crate chrono_tz;
+/// use chrono::{Duration, Offset, TimeZone};
+/// use chrono_tz::Europe::London;
+/// use chrono_tz::OffsetName;
+///
+/// # fn main() {
+/// let london_time = London.ymd(2016, 2, 10).and_hms(12, 0, 0);
+/// assert_eq!(london_time.offset().tz_id(), "Europe/London");
+/// // London is normally on GMT
+/// assert_eq!(london_time.offset().abbreviation(), "GMT");
+///
+/// let london_summer_time = London.ymd(2016, 5, 10).and_hms(12, 0, 0);
+/// // The TZ ID remains constant year round
+/// assert_eq!(london_summer_time.offset().tz_id(), "Europe/London");
+/// // During the summer, this becomes British Summer Time
+/// assert_eq!(london_summer_time.offset().abbreviation(), "BST");
+/// # }
+/// ```
+pub trait OffsetName {
+    /// The IANA TZDB identifier (ex: America/New_York)
+    fn tz_id(&self) -> &str;
+    /// The abbreviation to use in a longer timestamp (ex: EST), taking into
+    /// account any special offsets that may be in effect. For example, at a given instant,
+    /// the time zone with ID America/New_York may be either EST or EDT.
```

Comment:
Could you reformat this so that it has a separate subject/explanation line in the online
docs, something like:

```suggestion
    /// The abbreviation to use in a longer timestamp (ex: EST)
    ///
    /// This takes into account any special offsets that may be in effect.
    /// For example, at a given instant, the time zone with ID *America/New_York*
    /// may be either *EST* or *EDT*.
```

Reply:
Good call.